servers/parser-graphql-wrapper/src/test/utils/parserResponse.ts

```diff
@@ -0,0 +1,267 @@
+import nock, { Scope } from 'nock';
+import config from '../../config';
+import {
+  BoolStringParam,
+  MediaTypeParam,
+  ParserAPI,
+  ParserAPIOptions,
+  ParserAPIQueryParams,
+} from '../../datasources/ParserAPI';
+import { ParserResponse } from '../../datasources/ParserAPITypes';
+import { faker } from '@faker-js/faker';
+import { Videoness } from '../../__generated__/resolvers-types';
+import { merge } from 'lodash';
+import Keyv from 'keyv';
+
+export const fakeArticle: Partial<ParserResponse> = {
+  isArticle: 1,
+  article: `<p>"Not so handsome now", thought Harry as he dipped Hermione in hot sauce. The Death Eaters were dead now, and Harry was hungier than he'd ever been.</p>`,
```

Comment:
it would have been a tragedy to replace this data

Reply:
yea it really would have